### ISSUE TITLE
Translate wp_terms_checklist

### ISSUE THREAD

**Issue by xabispacebiker** [qTranslate-Team/qtranslate-x#629]
By now wp_terms_checklist are not translated so the users get something like this:

![Captura de pantalla 2020-10-08 a las 23 16 36](https://user-images.githubusercontent.com/1733884/95514532-63ee5f00-09bc-11eb-8ac7-f91da9962061.png)

Would it be possible to get all those terms translated?

**Comment by abclution** [comment]
https://github.com/qtranslate/qtranslate-xt

Qtranslate-x is abandoned for many years.
There is new modern versions, qtranslate-xt available at that repo.

**Comment by xabispacebiker** [comment]
> https://github.com/qtranslate/qtranslate-xt
> 
> Qtranslate-x is abandoned for many years.
> There is new modern versions, qtranslate-xt available at that repo.

Yep, I saw it.. that's why i open a new issue there https://github.com/qtranslate/qtranslate-xt/issues/902

Thanks!

**Comment by abclution** [comment]
Very welcome!
